config/environments/development.rb

```diff
@@ -68,6 +68,5 @@
 
   config.action_mailer.default_url_options = { host: (ENV["DOMAIN"] || "bops-care.link"), port: 3000 }
 
-  config.hosts << ".bops-care.link"
-  config.hosts << "southwark.southwark.web"
+  config.hosts.clear
```

Comment:
I keep having to do this for the local environment to work, should I separate it into its own commit and pop it outta here?

Reply:
I think this was fixed in the latest 6.1 release? I remember seeing something about `config.hosts` starting with `.`

Reply:
it only started happening after the 6.1 release, now no matter what I put in config.hosts it asks me to add it to config.hosts - unless I call config.hosts.clear it doesn't work locally 🤷